packages/sprotty/package.json

```diff
@@ -23,8 +23,10 @@
   },
   "dependencies": {
     "autocompleter": "^9.1.2",
+    "commander": "^12.0.0",
     "file-saver": "^2.0.5",
     "inversify": "~6.0.2",
+    "puppeteer": "^22.2.0",
```

Comment:
We should not have these dependencies in the base package of Sprotty. If we go for this solution, the new code should be moved to a new package.